src/benchmarkstt/normalization/__init__.py

```diff
@@ -1,99 +1,34 @@
-from collections import namedtuple
-import inspect
-from importlib import import_module
-from typing import Dict
-from benchmarkstt.docblock import format_docs
+from benchmarkstt.normalization.logger import log
+import logging
+from benchmarkstt.factory import Factory
 
 _normalizer_namespaces = (
     "benchmarkstt.normalization.core",
     ""
 )
 
-NormalizerConfig = namedtuple('NormalizerConfig', ['name', 'cls', 'docs', 'optional_args', 'required_args'])
 
+logger = logging.getLogger(__name__)
 
-def is_normalizer(cls):
-    return inspect.isclass(cls) and hasattr(cls, 'normalize')
 
+class Base:
```

Comment:
I've taken the design decision to name them `Base` under their respective subpackages, this is kind of a pseudo-namespace, i.e. we use `normalization.Base`, `metrics.Base`, etc. 

The re-use of the simple name `Base` allowed some easy abstracting away for the `Factory`, but this is no longer needed since we provide the class in the __init__ call of Factory. I am very much willing to change these `Base` names to something more descriptive if this is the preference.

Reply:
Ah yes, after I made this comment I noticed `Base` was being used in other classes too. No need to change it.